backend/util.js

```diff
@@ -10,14 +10,15 @@ const BASICS = [
 ];
 
 function controlCubeSettingsAndTransformList(cube, seats, type) {
-  let {list, cards, packs, cubePoolSize} = cube;
+  let {list, cards, packs, cubePoolSize, burnsPerPack} = cube;
 
   assert(typeof list === "string", "cube.list must be a string");
   assert(typeof cards === "number", "cube.cards must be a number");
   assert(typeof cubePoolSize === "number", "cube.cubePoolSize must be a number");
   assert(5 <= cards && cards <= 30, "cube.cards range must be between 5 and 30");
   assert(typeof packs === "number", "cube.packs must be a number");
   assert(1 <= packs && packs <= 12, "cube.packs range must be between 1 and 12");
+  assert(0 <= burnsPerPack && burnsPerPack <= 4, "cube.burnsPerPack range must be between 0 and 2");
```

Comment:
Your string says "between 0 and 2" but your assertion is 0 and 4

Reply:
Thanks for the review. I just corrected it.